packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec

```diff
@@ -71,19 +78,24 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 
 %files
 %dir %{gem_instdir}
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/discovery.yml
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/bundler.d
 %{gem_libdir}
-%{gem_spec}
-%{foreman_proxy_bundlerd_dir}/discovery.rb
-%config(noreplace) %{foreman_proxy_settingsd_dir}/discovery.yml
-%doc %{gem_instdir}/LICENSE
+%{gem_instdir}/settings.d
```

Comment:
Probably not needed

Reply:
This comes directly from `gem2rpm`s `main_file_entries` which lists everything from the gems `files` list, so I'm not sure how to tell it not to add it here

Reply:
I dropped these from this spec file for now.